druid-shell/src/platform/windows/window.rs

```diff
@@ -1682,12 +1730,16 @@ impl WindowHandle {
         if let Some(w) = self.state.upgrade() {
             let hwnd = w.hwnd.get();
             unsafe {
-                let show = match self.get_window_state() {
-                    window::WindowState::Maximized => SW_MAXIMIZE,
-                    window::WindowState::Minimized => SW_MINIMIZE,
-                    _ => SW_SHOWNORMAL,
-                };
-                ShowWindow(hwnd, show);
+                if w.is_focusable {
+                    let show = match self.get_window_state() {
+                        window::WindowState::Maximized => SW_MAXIMIZE,
+                        window::WindowState::Minimized => SW_MINIMIZE,
+                        _ => SW_SHOWNORMAL,
+                    };
+                    ShowWindow(hwnd, show);
+                } else {
+                    ShowWindow(hwnd, SW_SHOWNOACTIVATE);
+                }
```

Comment:
If you say
```rust
let show = if w.is_focusable { ...
} else {
  SW_SHOWNOACTIVATE
};
```
Then you can have the `unsafe` just around the `ShowWindow`...

Reply:
Fixed as suggested, and other occurrence on L592-L600.

Also added entry in CHANGELOG and fixed 2-3 lint errors in there, I hope that's OK @jneem?

Reply:
Sounds good to me!